src/app/personside/infotabs/oversikt/YtelserOversikt.tsx

```diff
@@ -43,53 +44,59 @@ function YtelserOversikt() {
 function PleiepengerKomponent(props: { pleiepenger: Pleiepengerettighet }) {
     const dyplenker = useInfotabsDyplenker();
     return (
-        <VisMerKnapp
-            linkTo={dyplenker.ytelser.link(getUnikPleiepengerKey(props.pleiepenger))}
-            valgt={false}
-            ariaDescription="Vis pleiepenger"
-        >
-            <Normaltekst>
-                <Bold>Pleiepenger sykt barn</Bold>
-            </Normaltekst>
-            <Normaltekst>Barnets f.nr: {props.pleiepenger.barnet}</Normaltekst>
-        </VisMerKnapp>
+        <div className={ytelserTest.oversikt}>
```

Comment:
litt dumt å endre på dom'en for å lage en test..? Kan du sette klassenavnet på VisMerKnapp istedenfor?

Reply:
Tidligere har vi lagt classname i en <li> ettersom VisMerKnapp ikke tar en className. Vi kan bruke fredagen til å se litt om vi kan legge alle Ytelser inn i en komponent, istedenfor tre separate